### PR TITLE
Standardization/integration radio

### PR DESCRIPTION
**Pull request by virgotopia**

### CODE REVIEW

File: client/app/domain/redirection/add/domain-redirection-add.html [hunk 1]
```diff
@@ -43,33 +43,26 @@
...
-                        <span class="oui-radio__description"
-                              data-ng-bind-html="i18n.domain_tab_REDIRECTION_add_step2_server_help"></span>
-                    </label>
-                </div>
+                <oui-radio-group data-ng-init="ctrl.newRedirection.step2 = ctrl.choice.web"
+                        value="ctrl.newRedirection.step2"
+                        on-change="ctrl.setTargetRedirection($event)">
+
+                    <!-- to web address -->    
+                    <oui-radio label="{{tr('domain_tab_REDIRECTION_add_step2_web')}}"
```
Comment (AxelPeter) [inline client/app/domain/redirection/add/domain-redirection-add.html]: Bad attribute, it's `text="{{::tr('domain_tab_REDIRECTION_add_step2_web')}}"`. You can add too a one time-binding with `::`.

See http://master.ui-kit.ovh/#!/oui-angular/radio :)

File: client/app/domain/redirection/add/domain-redirection-add.html [hunk 2]
```diff
@@ -43,33 +43,26 @@
...
+                               value="{{ctrl.choice.web}}">
```
Comment (AxelPeter) [inline client/app/domain/redirection/add/domain-redirection-add.html]: You have to remove the brackets -> `value="ctrl.choice.web"`

File: client/app/domain/redirection/add/domain-redirection-add.html [hunk 3]
```diff
@@ -43,33 +43,26 @@
...
+                        <oui-radio-description>
```
Comment (AxelPeter) [inline client/app/domain/redirection/add/domain-redirection-add.html]: The description is provided by an attribute, `description="{{::i18n.domain_tab_REDIRECTION_add_step2_web_help}}"`

File: client/app/domain/redirection/add/domain-redirection-add.html [hunk 4]
```diff
@@ -43,33 +43,26 @@
...
-                           data-ng-model="ctrl.newRedirection.step2"
-                           data-ng-value="ctrl.choice.server">
-                    <label class="oui-radio__label-container" for="choiceTargetRedirectionServer">
-                        <span class="oui-radio__label"
-                              data-i18n-static="domain_tab_REDIRECTION_add_step2_server"></span>
-                        <span class="oui-radio__description"
-                              data-ng-bind-html="i18n.domain_tab_REDIRECTION_add_step2_server_help"></span>
-                    </label>
-                </div>
+                <oui-radio-group data-ng-init="ctrl.newRedirection.step2 = ctrl.choice.web"
```
Comment (AxelPeter) [inline client/app/domain/redirection/add/domain-redirection-add.html]: The component `<oui-radio-group>` doesn't work for the moment.
http://master.ui-kit.ovh/#!/oui-angular/radio-group is broken :(

Do only with http://master.ui-kit.ovh/#!/oui-angular/radio for now ;)

File: client/app/website/configuration.html [hunk 5]
```diff
@@ -15,34 +15,33 @@ <h1 data-i18n-static="website_title"></h1>
...
+                        description="{{::tr('website_creation_type_classic_explanation')}}"
+                        on-change="webSiteConfigurationCtrl.site.config.type = 'classic'"
+                        tag-click tag-name="website_configuration_type_classic"
+                        model="webSiteConfigurationCtrl.site.config.type"
+                        name="website-configuration"
+                        value="'classic'" checked>                       
+                </oui-radio>
+
+                <oui-radio text="{{::tr('website_creation_type_blog')}}"
+                        description="{{::tr('website_creation_type_blog_explanation')}}"
```
Comment (FredericEspiau) [inline client/app/website/configuration.html]: We should use `data-` for non HTML 5 properties

File: client/app/domain/zone/record/SPF/domain-zone-record-SPF.html [hunk 6]
```diff
@@ -148,33 +142,25 @@
...
-                       data-ng-change="ctrl.setTargetValue('SPF')"
-                       data-ng-model="ctrl.model.target.all">
-                <label class="oui-radio__label-container" for="all-no">
-                    <span class="oui-radio__label"
-                          data-i18n-static="domain_configuration_dns_entry_add_spf_all_title_no"></span>
-                </label>
-            </div>
+
+            <oui-radio text="{{::tr('domain_configuration_dns_entry_add_spf_all_title_yes')}}"
+                    on-change="ctrl.model.target.all = '-all'; ctrl.setTargetValue('SPF')"
```
Comment (FredericEspiau) [inline client/app/domain/zone/record/SPF/domain-zone-record-SPF.html]: `ctrl.changeTarget('-')` ?

File: client/app/domain/zone/record/SPF/domain-zone-record-SPF.html [hunk 7]
```diff
@@ -148,33 +142,25 @@
...
+                    model="ctrl.model.target.all"
+                    name="spf-all"
+                    value="'-all'">
+            </oui-radio>
+            <oui-radio text="{{::tr('domain_configuration_dns_entry_add_spf_all_title_safe')}}"
+                    on-change="ctrl.model.target.all = '~all'; ctrl.setTargetValue('SPF')"
```
Comment (FredericEspiau) [inline client/app/domain/zone/record/SPF/domain-zone-record-SPF.html]: `ctrl.changeTarget('~')` ?

File: client/app/domain/zone/record/SPF/domain-zone-record-SPF.html [hunk 8]
```diff
@@ -148,33 +142,25 @@
...
+                    model="ctrl.model.target.all"
+                    name="spf-all"
+                    value="'~all'">
+            </oui-radio>
+            <oui-radio text="{{::tr('domain_configuration_dns_entry_add_spf_all_title_no')}}"
+                    on-change="ctrl.model.target.all = '?all'; ctrl.setTargetValue('SPF')"
```
Comment (FredericEspiau) [inline client/app/domain/zone/record/SPF/domain-zone-record-SPF.html]: `ctrl.changeTarget('?')` ?

File: client/app/domain/zone/record/SPF/domain-zone-record-SPF.html [hunk 9]
```diff
@@ -13,77 +13,71 @@
...
-                       data-ng-model="ctrl.model.target.ptrSender"
-                       data-ng-value="true">
-                <label class="oui-radio__label-container" for="ptrSender-yes">
-                    <span class="oui-radio__label"
-                          data-i18n-static="common_yes"></span>
-                </label>
+
+            <div class="oui-radio_inline">
+                <oui-radio text="{{::tr('common_yes')}}"
+                        on-change="ctrl.model.target.ptrSender = true; ctrl.setTargetValue('SPF')"
```
Comment (FredericEspiau) [inline client/app/domain/zone/record/SPF/domain-zone-record-SPF.html]: Maybe you could find a way to have less redundancy ?

**Comment by AxelPeter** [comment]
Currently, not a priority. We'll reopen it when the update v2, navbar and datagrids are done.

**Pull request closed without merging**